Cargo.toml

```
@@ -9,6 +9,7 @@ anyhow = "1.0"
 async-trait = "0.1"
 futures = "0.3"
 primitive-types = { version = "0.10", features = ["fp-conversion"], optional = true }
+ethcontract = { git = "https://github.com/gnosis/ethcontract-rs.git", branch = "eip1559", features = ["gas_price"], optional = true}
```

```suggestion
ethcontract = { git = "https://github.com/gnosis/ethcontract-rs.git", branch = "eip1559", optional = true}
```

I don't see a `gas-price` feature in [`ethcontract`](https://github.com/gnosis/ethcontract-rs/blob/eip1559/ethcontract/Cargo.toml). Am I missing something?

---

It's here: https://github.com/gnosis/ethcontract-rs/pull/700
This PR is dependent on it.